terraform_test.tf

```diff
@@ -116,4 +116,7 @@ resource "maas_instance" "tf_test_vm_instance" {
   deploy_params {
     distro_series = var.distro_series
   }
+  timeout {
```

Comment:
This timeout should at least be set to `maas_vm_host` resource above. This is what is causing our tests to fail